demo/template.marko

```diff
@@ -17,7 +17,18 @@
             <div class="example">
                 <h4>${example.name}</h4>
                 <div class="example-code">$!{example.code}</div>
-                <div><dynamic-tag options={path: example.path}/></div>
+                <div>
+                    <!-- icons need to be statically imported -->
+                    <if(data.params.component === 'ebay-icon')>
```

Comment:
Is this still needed?

Reply:
It looks like this is still needed, because the icons are required too late to be used by the stamper. Theoretically it would be ok after the first request I think, but those don't work either because we're caching the page's icons on the first request.

I'll have to do this static import for our internal components that use inline `<ebay-icon>` too.